README.md

```diff
@@ -114,6 +114,9 @@ We also have explicit _non-goals_ for Carbon, notably including:
 
 -   A stable ABI for the entire language and library
 -   Perfect backwards or forwards compatibility
+-   Support legacy code for which source code is unavailable
```

Comment:
I think this is tricky to include.

I think it is a non-goal to fully support such code at 100% fidelity, ergonomics, and performance in our interop story.

But I think it is a *goal* to have *some* support story for such code, which generally isn't a performance regression from C++ and is sufficient for the Carbon users that need to cope with such code.

Unfortunately, trying to distill that into a single bullet point I think loses the important nuance and will be likely to read as "absolutely zero support" which isn't (IMO) accurate.